src/main/java/com/jetblue/task/fetch/route/dao/RouteDao.java

```diff
@@ -0,0 +1,11 @@
+package com.jetblue.task.fetch.route.dao;
+
+import com.jetblue.task.fetch.route.model.Route;
+
+import java.util.List;
+
+public interface RouteDao {
```

Comment:
Normally we are not creating interface for Dao layer in JetBlue projects

Reply:
okay